cli.js

```diff
@@ -25,31 +24,10 @@ if (cli.input.length === 0) {
 	process.exit(1);
 }
 
-var files = [];
-
-eachAsync(cli.input, function (el, i, cb) {
-	pathExists(el, function (err, exists) {
-		if (err) {
-			cb(err);
-			return;
-		}
-
-		if (exists) {
-			files.push(el);
-		}
-
-		cb();
-	});
-}, function (err) {
-	if (err) {
-		console.error(err.message);
-		process.exit(1);
-	}
-
-	trash(files, function (err) {
-		if (err) {
-			console.error(err.message);
-			process.exit(1);
-		}
+Promise.all(cli.input.map(function (el) {
+	return pify(pathExists, Promise)(el).then(function (exists) {
```

Comment:
`path-exists` has already been updated to use a Promise API so I don't think there's a reason to `pify` it?